lib/glimesh_web/live/user_live/stream.ex

```diff
@@ -6,7 +6,7 @@ defmodule GlimeshWeb.UserLive.Stream do
   alias Glimesh.Presence
 
   def mount(%{"username" => streamer_username}, session, socket) do
-    case Streams.get_by_username(streamer_username) do
+    case Streams.get_by_username(String.downcase(streamer_username)) do
```

Comment:
Do we need the downcase here?

Reply:
Nope, will fix in next commit. Couldn't remember where I put those.